styles/assessment.module.css

```diff
@@ -11,6 +11,7 @@
   display: flex;
   flex-direction: column;
   align-items: center;
+  /* background-color: aquamarine; */
```

Comment:
We need to remember to remove these at some point

Reply:
Oops, I can remove them now